lib/roo/excelx/cell.rb

```diff
@@ -40,19 +40,24 @@ def type
       end
 
       def self.create_cell(type, *values)
```

Comment:
I've avoided using create_cell as calling this method will create intermediate array object for values. Generally, I'll be against doing this but this method is invoked once per cell which can result in noticeable overhead.